vmdb/app/models/customization_script.rb

```diff
@@ -1,4 +1,7 @@
 class CustomizationScript < ActiveRecord::Base
   include NewWithTypeStiMixin
   belongs_to :provisioning_manager
+
+  scope :ptables, where(:type => "CustomizationScriptPtable")
+  scope :media, where(:type => "CustomizationScriptMedium")
```

Comment:
The scopes are weird to begin with...why would a caller need to do `CustomizationScript.ptables.where()` when they can just do `CustomizationScriptPtable.where()`